cloudbridge/providers/gcp/provider.py

```diff
@@ -336,17 +343,33 @@ def sign_blob(self, string_to_sign):
     def client_id(self):
         return self._credentials.service_account_email
 
+    def _get_build_request(self):
+        credentials = Credentials.from_service_account_info(self.credentials_dict)
+        credentials=with_scopes_if_required(credentials, list(CLOUD_SCOPES))
```

Comment:
```suggestion
        credentials = with_scopes_if_required(credentials, list(CLOUD_SCOPES))
```

Reply:
@nuwang I'm probably missing something but where is `credentials` defined in your edit?

Reply:
@selshowk Sorry, my mistake, I thought the second credentials was overwriting the first - it just needs spacing.